docs/installation/configuration/submission.md

```diff
@@ -57,3 +57,46 @@ Since this section is quite simple, we will list the default configuration at th
 
 !!! tip
     Refer to the [changing the configuration file](../config_file/#changing-the-configuration-file) documentation for more detail on where and how to change the configuration of the system.
+
+## Metadata Validation
+
+You can configure the system to enforce metadata validation and presence when performing ingestion and archiving. This is a useful feature if you're looking to harmonize the metadata from different sensors under a common scheme.
```

Comment:
```suggestion
You can configure the system to enforce metadata validation and presence when performing ingestion and archiving. This is a useful feature if you're looking to harmonize the metadata from different sources under a common scheme.
```
Not a mandatory change, I just thought it would be a more generic word, in case someone had feeds from VT and MB, and wants to normalize them.